docs/src/man/alignments.md

```diff
@@ -440,3 +440,87 @@ julia> submat['A','B']  # mismatch
 -1
 
 ```
+
+
+## High-throughput sequencing file formats
+
+One of the major sources to generate massive amounts of sequence alignments is
+high-throughput sequencing technologies. The `Bio.Align` module provides several
+data formats commonly used in this field.
+
+
+### SAM and BAM file formats
+
+SAM and BAM would be the most popular file formats and their readers have the
+same interface as other readers in Bio.jl:
```

Comment:
Change to "BAM and SAM formats have the same reading and writing interface as all other formats in Bio.jl" with a ref to the section.